andela/Titan-Devs
# #163518703 Like Comment

#### What does this PR do?
- It helps the logged in user to like a specific comment
#### Description of Task to be completed?
- Write the tests for liking a comment on an article
- Fetch users who liked a comment

#### How should this be manually tested?
- After cloning the repository, check out `ft-like-comment-163518703` branch
- Make sure to read `README.md` file to get the setup instructions
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Create an article using `POST /api/v1/articles` endpoint
- Comment on the created article using `POST /api/v1/articles/:slug/comments` endpoint
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
- The request's body should be as mentioned in the `README.md` file under `Endpoints:` ==> `Add Comments to an Article` subtitle
- After the successful comment creation, you should get back comment information. 
- Grab the comment Id and it use to consume `POST  /api/v1/articles/:slug/comments/<commentId>likes`. You shall see a `Comment liked message response` with the liked comment. If you try to like a comment twice, with the same access token, you shall see a `Comment unliked message` with the decremented amount of likes on a comment. 
- Consume the `GET  /api/v1/articles/:slug/comments/<commentId>likes` endpoint. You shall get a comment with the `like` and `likedBy` attributes. 

#### What are the relevant pivotal tracker stories?
- #63518703
